scala/scala.bzl

```diff
@@ -782,10 +782,9 @@ _common_attrs_for_plugin_bootstrapping = {
 }
 
 _common_attrs = _common_attrs_for_plugin_bootstrapping + {
-  # dependency_analyzer_mode_soon_to_be_removed will be replaced by using command line flag called 'strict_java_deps' (https://github.com/bazelbuild/bazel/issues/3295)
+  # using stricts scala deps is done by using command line flag called 'strict_java_deps' 
```

Comment:
I agree but since this is not really documented (i.e. not in readme) I'm good with this ambiguity.